client/components/mma/paymentUpdate/card/FlexCardElement.tsx

```diff
@@ -78,22 +80,26 @@ export const FlexCardElement = (props: FlexCardElementProps) => (
 				`}
 			>
 				<FieldWrapper width="50%" label="Expiry Date">
-					<CardExpiryElement
-						options={{
-							style: baseStyle,
-							placeholder: 'MM/YY',
-						}}
-						onReady={props.setCardExpiryElement}
-					/>
+					<span data-qm-masking="blocklist">
```

Comment:
do we need the `span` or could this go in the `FieldWrapper` element?

Reply:
I did try it in the `FieldWrapper` and it didn't work which is why I added the `span`